### DISCUSSION TITLE
How to persist image transparency?

### DISCUSSION THREAD

**Discussion by Richard-Supe** [ValentinH/react-easy-crop#462] [Q&A] [answered]
This is a repeat of https://github.com/ricardo-ch/react-easy-crop/issues/308 which no longer exists. I'd like to maintain uploaded image transparent background, or at least be able to specify the background color.

Currently playing around with fillColor on the HTMLCanvasElement, but not sure if this is the best approach

**Comment by ValentinH** [comment, marked as answer]
You should set the exported image to be a png in the crop image.js file.

See #308. Too bad that Github stopped automatically redirecting from the old organisation 🙈

**Reply by Richard-Supe** [reply]
Awesome, Thanks!
